packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -313,42 +313,39 @@ const addScaffoldImport = () => {
   return 'Added scaffold import to index.js'
 }
 
-export const command = 'scaffold <pathSlashModel>'
-export const desc = 'Generate pages, SDL, and a services object.'
-export const builder = {
-  force: { type: 'boolean', default: false },
-  // So the user can specify a path to nest the generated files under.
-  // E.g. yarn rw g scaffold post --path=admin
-  path: { type: 'string', default: false },
+export const command = 'scaffold <model>'
+export const desc =
+  "Generate Pages, SDL, and Services files based on a given DB schema Model. Also accepts <path/model> (see 'scaffold --help')."
+export const builder = (yargs) => {
+  yargs.positional('model', {
+    description:
+      'Model to scaffold. You can also use <path/model> to nest files by type at the given path directory (or directories)--e.g. admin/post',
```

Comment:
`Model to scaffold. You can also use <path/model> to nest files by type at the given path directory (or directories). For example, 'rw g scaffold admin/post'.`

y/n/maybe?

Reply:
That looks good to me. Is it worth being extra explicit like:

```diff
Model to scaffold. You can also use <path/model> to nest files by type at the 
given path directory (or directories). 
- For example, 'rw g scaffold admin/post'.
+ For example, 'rw g scaffold admin/post' where post is a model 
+ in your schema.prisma.
```

It's definitely getting long at that point.

Reply:
Changed it to your suggestion for the time being.

Reply:
At this level of --help (meaning being very zoomed in), I don't think that's too much text. So go for it if you want to include `For example, 'rw g scaffold admin/post' where post is a model in your schema.prisma.`

Reply:
Going to stick with your suggestion; we say `<path/model>` before and then give the example `'admin/post'` so I think that should be enough.